pkg/virt-handler/virtwrap/api/schema.go

```diff
@@ -142,15 +142,31 @@ type DomainList struct {
 }
 
 type DomainSpec struct {
-	XMLName xml.Name `xml:"domain"`
-	Name    string   `xml:"name"`
-	UUID    string   `xml:"uuid,omitempty"`
-	Memory  Memory   `xml:"memory"`
-	Type    string   `xml:"type,attr"`
-	OS      OS       `xml:"os"`
-	SysInfo *SysInfo `xml:"sysinfo,omitempty"`
-	Devices Devices  `xml:"devices"`
-	Clock   *Clock   `xml:"clock,omitempty"`
+	XMLName  xml.Name     `xml:"domain"`
+	Type     string       `xml:"type,attr"`
+	XmlNS    string       `xml:"xmlns:qemu,attr,omitempty"`
+	Name     string       `xml:"name"`
+	UUID     string       `xml:"uuid,omitempty"`
+	Memory   Memory       `xml:"memory"`
+	OS       OS           `xml:"os"`
+	SysInfo  *SysInfo     `xml:"sysinfo,omitempty"`
+	Devices  Devices      `xml:"devices"`
+	Clock    *Clock       `xml:"clock,omitempty"`
+	Resource *Resource    `xml:"resource,omitempty"`
```

Comment:
Is there a bug or RFE filed against libvirt to improve the resource field so we can use it?